web-common/src/features/dashboards/url-state/convertExploreStateToURLSearchParams.ts

```diff
@@ -279,6 +279,16 @@ function toExploreUrl(
     );
   }
 
+  const leaderboardMeasuresParam = toLeaderboardMeasuresUrlParam(exploreState);
+  if (leaderboardMeasuresParam !== undefined) {
+    searchParams.set(
+      ExploreStateURLParams.LeaderboardMeasures,
+      leaderboardMeasuresParam,
+    );
+  } else {
+    searchParams.delete(ExploreStateURLParams.LeaderboardMeasures);
```

Comment:
Curious, do we have to delete the param here? Feels like it shouldn't be necessary.

Reply:
are you aware of other code abstractions to better handle this? @AdityaHegde

Reply:
removing since I don't see the previous behavior anymore

Reply:
When I manually remove a measure from `leadeboard_measures`, I get the behavior I described. See:

https://github.com/user-attachments/assets/fbdb2a93-878c-4e71-9626-b04a35f48b61